### DISCUSSION TITLE
Add back 'null' as type

### DISCUSSION THREAD

**Discussion by silveltman** [joe-bell/cva#236] [Q&A]
With V1 I'm aware that null is removed as a possible type with VariantProps. However, I would like to re-add it.

I set component variants in my headless CMS (payload), which also generates types for me. However, it adds null to non-required types.

null would not have to have any functionality in my case, just not raise errors.

**Comment by joe-bell** [comment]
Sorry! I don't have any plans to reintroduce

**Reply by silveltman** [reply]
I understand! Is there a way to manually add a this myself? 

I looked into 'unset' but don't totally understand it
